### ISSUE TITLE
update jruby version to 1.7.27 on 5.x releases

### ISSUE THREAD

**Issue by jsvd** [elastic/logstash#7477]
TL;DR - **Using jruby 1.7.27 in logstash 5.x solves the entropy issues. tested myself and went from 5 minutes to 20 seconds of time to startup**

Other than the "we should use the latest version of jruby", this version fixes the well known issue of entropy causing extremely long start up times, described in https://github.com/elastic/logstash/issues/5507

I used a t2.micro and depleted the entropy pool in a separate terminal with `cat /dev/random`, then I confirmed things take a long time to start (logstash 5.4.1 uses jruby 1.7.25):

```
logstash-5.4.1$ bin/ruby -e 'puts RUBY_DESCRIPTION'
jruby 1.7.25 (1.9.3p551) 2016-04-13 867cb81 on OpenJDK 64-Bit Server VM 1.8.0_131-b12 +jit [linux-amd64]

logstash-5.4.1$ time bin/logstash -e "input {} output {}"
[2017-06-16T12:25:57,234][INFO ][logstash.pipeline        ] Starting pipeline {"id"=>"main", "pipeline.workers"=>1, "pipeline.batch.size"=>125, "pipeline.batch.delay"=>5, "pipeline.max_inflight"=>125}
[2017-06-16T12:25:57,275][INFO ][logstash.pipeline        ] Pipeline main started
[2017-06-16T12:25:57,491][INFO ][logstash.agent           ] Successfully started Logstash API endpoint {:port=>9600}
[2017-06-16T12:26:00,444][WARN ][logstash.agent           ] stopping pipeline {:id=>"main"}

real	5m13.083s
user	0m18.240s
sys	0m0.515s
```

then I took logstash 5.4 branch and built a tar release with jruby 1.7.27:

```
logstash-5.4.2-SNAPSHOT$ bin/ruby -e 'puts RUBY_DESCRIPTION'
jruby 1.7.27 (1.9.3p551) 2017-05-11 8cdb01a on OpenJDK 64-Bit Server VM 1.8.0_131-b12 +jit [linux-amd64]
```

Then performed the same test:

```
logstash-5.4.2-SNAPSHOT $ time bin/logstash -e "input {} output {}"
[2017-06-16T12:20:28,859][INFO ][logstash.pipeline        ] Starting pipeline {"id"=>"main", "pipeline.workers"=>1, "pipeline.batch.size"=>125, "pipeline.batch.delay"=>5, "pipeline.max_inflight"=>125}
[2017-06-16T12:20:28,880][INFO ][logstash.pipeline        ] Pipeline main started
[2017-06-16T12:20:29,127][INFO ][logstash.agent           ] Successfully started Logstash API endpoint {:port=>9600}
[2017-06-16T12:20:32,087][WARN ][logstash.agent           ] stopping pipeline {:id=>"main"}

real	0m19.866s
user	0m16.670s
sys	0m0.467s
```

**Comment by ph** [comment]
#7478 was merged we can close this.
